rev wording describing color lines (#292)
googlefonts/colr-gradients-spec: OFF_AMD2_WD.md

If only one color stop is specified, that color is used for the entire color
line; at least two color stops are needed to create color gradation.

Color gradation is defined over the interval from the color stop with the
minimum offset, through successive color stops, to the color stop with the
maximum offset. Between numerically-adjacent color stops, color values are
linearly interpolated. See _Interpolation of Colors_ in 5.7.12 for requirements
on how colors are interpolated.

Color values outside the defined interval are determined by the color line’s
*extend mode*, described below. In this way, colors are defined for all stop
offset values, from negative infinity to positive infinity.

For example, a gradient color line could be defined with two color stops at 0.2
and 1.5. Colors for offsets between 0.2 and 1.5 are interpolated. Colors for
offsets above 1.5 and below 0.2 are determined by the color line’s extend mode.
The gradient color line is positioned in the design grid by aligning stop

> offsets 0 and 1 to design grid positions.

Reviewer: "design grid positions" mean "point 0" and "point 1" in the case of a linear gradient, right? For sweep gradients it's about angles, and the extend mode works differently if I understand correctly. For radial gradients, it's about the center points *and* the radii, correct?

In other words: The phrase "design grid positions" confused me because it's not explicit what it means for linear gradients, and is more unclear what it means for radial gradients, and is most confusing for sweep gradients.

Reviewer 2: I believe we clarify this per-type, would it suffice to delete "The gradient color line is positioned in the design grid by aligning stopoffsets 0 and 1 to design grid positions." entirely?

color stops, color values are linearly interpolated. See _Interpolation of
Colors_ in 5.7.12 for requirements on how colors are interpolated.

> While the color gradation is specified over a defined interval, the color line
> continues indefinitely outside that interval in both directions. The color
> patterns outside the defined interval are determined by the color line’s *extend
> mode*, described below. In this way, colors are defined for all stop offset
> values, from negative infinity to positive infinity.

Reviewer 3: Perhaps shortening both paragraphs (this and the previous), avoiding redundancy and talking about extrema as references helps to make clear that the minimum and maximum define the range and that 0 and 1 do not play a role? 

> Color gradation is defined over the interval from the first color stop, through
> the successive color stops, to the last color stop. Between numerically-adjacent
> color stops, color values are linearly interpolated. See _Interpolation of
> Colors_ in 5.7.12 for requirements on how colors are interpolated.

=>

Color gradation is defined over the interval from the minimum color stop, through
the successive color stops, to the maximum color stop. Between numerically-adjacent
color stops, color values are linearly interpolated. See _Interpolation of
Colors_ in 5.7.12 for requirements on how colors are interpolated.

Then shorter

```suggestion
The color
values for positions outside the defined interval from minimum to maximum are determined by the color line’s *extend
mode*, described below. In this way, colors are defined for all stop offset values, from negative infinity to positive infinity. 
```

For example, a gradient color line could be defined with two color stops at 0.2
and 1.5. Colors for offsets between 0.2 and 1.5 are interpolated. Colors for
offsets above 1.5 and below 0.2 are determined by the color line’s extend mode.
The gradient color line is positioned in the design grid by aligning stop
offsets 0 and 1 to design grid positions. In this example, the color at stop
offset 1 will be derived by interpolation between the two color stops, and the

> color at stop offset 0 will be derived based on the extend mode.

Reviewer 3: Suggest to strip "In this example, the color at stop
offset 1 will be derived by interpolation between the two color stops, and the
color at stop offset 0 will be derived based on the extend mode." as color at stop offset 0 is derived by extend mode, but also by interpolation between _repeated_ color stops.
